example/docker/nginx-conf/nginx.conf

```diff
@@ -55,7 +55,7 @@ server {
     }
 
     # Public API
-    location /auth/api/seacat-auth {
+    location /auth/api/seacat-auth/public {
```

Comment:
is this correct?

Reply:
yes, Auth UI shouldn't access anything outside `/public` and `/openidconnect`.